tests/fixtures/tts.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+import soundfile as sf
+import torch
+
+from nemo.collections.asr.parts.utils.manifest_utils import read_manifest
+
+
+@pytest.fixture()
+def set_device():
+    return torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+
+@pytest.fixture()
+def language_specific_text_example():
+    return {
+        "en": "Caslon's type is clear and neat, and fairly well designed;",
+        "de": "Ich trinke gerne Kräutertee mit Lavendel.",
+        "es": "Los corazones de pollo son una delicia.",
+        "zh": "双辽境内除东辽河、西辽河等5条河流",
+    }
+
+
+@pytest.fixture()
+def supported_languages(language_specific_text_example):
+    return sorted(language_specific_text_example.keys())
+
+
+@pytest.fixture()
+def get_language_id_from_pretrained_model_name(supported_languages):
+    def _validate(pretrained_model_name):
+        language_id = pretrained_model_name.split("_")[1]
+        if language_id not in supported_languages:
+            pytest.fail(
+                f"`PretrainedModelInfo.pretrained_model_name={pretrained_model_name}` does not follow the naming "
+                f"convention as `tts_languageID_model_*`, or `languageID` is not supported in {supported_languages}."
+            )
+        return language_id
+
+    return _validate
+
+
+@pytest.fixture()
+def melSpec_example(set_device):
```

Comment:
Why melSpec_example instead of mel_spec_example?

Reply:
yeah, good question, I should change to `mel_spec_example`.